plugin/db/driver.go

```diff
@@ -387,6 +388,25 @@ type ConnectionContext struct {
 	InstanceName    string
 }
 
+// IncrementalRecoveryConfig defines a range in the incremental backup.
+type IncrementalRecoveryConfig struct {
+	// Start/End is the binlog position in MySQL, and WAL lsn in PostgreSQL.
+	// Encoded in JSON format.
+	Start []byte
```

Comment:
This isn't MySQL specific. I'm thinking of using the following. Btw, we should mention less about Incremental because it's not a canonical term.

And we don't need to encode json here, right? If that's for storage, it should be part of API package.

```
type RecoveryConfig struct {
  xx MySQLBinlogConfig
}
```